lib/howitzer/meta/iframe.rb

```diff
@@ -0,0 +1,53 @@
+module Howitzer
+  module Meta
+    # This class represents iframe entity within howitzer meta information interface
+    class Iframe
+      attr_reader :name, :context
+
+      def initialize(name, context)
+        @name = name
+        @context = context
+      end
+
+      # Finds all instances of iframe on the page and returns them as array of capybara elements
+      # @return [Array]
+      def capybara_elements
+        block = proc do |frame|
+          @site_value = frame.class.send(:site_value)
```

Comment:
Is method `site_value` private?
Can we call it directly `frame.class.site_value` ?

Reply:
Is it possible to ask `site_value` from frame class object like:
`context.send("#{name}_iframe").class.site_value`
or
`context.send("#{name}_iframe").class.send(:site_value)`?